.github/workflows/close-featurebranch.yml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
```

Comment:
Could we add `workflow_dispatch` to be able to manually delete branches. It might be useful for when something gets into a bad state (I did with my pipeline changes 😅 )

Reply:
Good shout! Done